options/linux/include/mlibc/linux-sysdeps.hpp

```diff
@@ -83,6 +84,12 @@ int sys_ioctl(int fd, unsigned long request, void *arg, int *result);
 
 [[gnu::weak]] int sys_getifaddrs(struct ifaddrs **);
 
+[[gnu::weak]] int sys_sendfile(int outfd, int infd, off_t *offset, size_t count, size_t *out);
```

Comment:
`sendfile()` returns `ssize_t`, not `size_t`. However, the value is always positive, so passing `size_t *` is probably okay. I'm not sure how we handle this for other syscalls, but if it's consistent with other calls, it can stay like this.

The same applies to `sys_splice()`.

Reply:
Fixed this at both places.